### ISSUE TITLE
start onvif_simple_server

### ISSUE THREAD

**Issue by onepif** [roleoroleo/onvif_simple_server#3]
Hello!
thank you for your work. Did I understand correctly that to start the onvif server it is enough to create a config in the etc folder with replacing the interface name and IP address and execute ./onvif_simple_server?

**Comment by roleoroleo** [comment]
No, it's not enough.
onvif_simple_server runs as a cgi binary, so you need to use a http server.
For example, in this project https://github.com/roleoroleo/yi-hack-Allwinner-v2 I use busybox httpd.

**Comment by roleoroleo** [comment]
If you want, I can share an example with lighttpd.

**Comment by onepif** [comment]
yes, if it’s not difficult, it’s advisable to have a step-by-step guide to start it up

**Comment by roleoroleo** [comment]
Check this script:
https://github.com/roleoroleo/onvif_simple_server/blob/master/extras/build.sh

**Comment by GiambaFr** [comment]
Hello, for sure an example with lighttpd would be appreciated. I don't know how to start the 3 CGIs inside the _install folder.

**Comment by roleoroleo** [comment]
When you run build.sh, the script will build a working example.
I added some note to the readme, check the last commit.
You don't need to run the cgi scripts manually.
The onvif client will call these scripts.
